Example how to add proxycert

vokac opened the pull request:
Example how user can call `/iam/account/me/proxycert` endpoint to store long lived X.509 proxy.

vokac committed:
Example how to add proxycert

Example how user can call `/iam/account/me/proxycert` endpoint to store long lived X.509 proxy.

federicaagostini commented:
Hi Petr, thank you for your PR.

Anyway, the `/iam/account/me/proxycert` endpoint has been described quickly [here](https://indigo-iam.github.io/v/v1.8.1/docs/reference/api/account-api/#put-iamaccountmeproxycert). Could you please extend that section with your example?

vokac commented:
OK, I missed this piece of documentation.

I'll try to update this pull request, but it might take me some time during busy week...

vokac committed:
Link IAM Account Proxy Certificate API

Provide link to to documentation that describes how to upload certificate to the user account.
